src/platform/plugins/shared/dashboard/public/dashboard_container/component/grid/dashboard_grid.tsx

```diff
@@ -7,153 +7,134 @@
  * License v3.0 only", or the "Server Side Public License, v 1".
  */
 
-import 'react-resizable/css/styles.css';
-import 'react-grid-layout/css/styles.css';
-
-import { pick } from 'lodash';
 import classNames from 'classnames';
-import React, { useState, useMemo, useCallback, useEffect } from 'react';
-import { Layout, Responsive as ResponsiveReactGridLayout } from 'react-grid-layout';
+import React, { useCallback, useMemo, useRef } from 'react';
 
-import { useBatchedPublishingSubjects } from '@kbn/presentation-publishing';
 import { useAppFixedViewport } from '@kbn/core-rendering-browser';
+import { ViewMode } from '@kbn/embeddable-plugin/public';
+import { GridLayout, type GridLayoutData } from '@kbn/grid-layout';
+
+import { useBatchedPublishingSubjects } from '@kbn/presentation-publishing';
 import { DashboardPanelState } from '../../../../common';
-import { DashboardGridItem } from './dashboard_grid_item';
-import { useDashboardGridSettings } from './use_dashboard_grid_settings';
-import { useDashboardApi } from '../../../dashboard_api/use_dashboard_api';
 import { arePanelLayoutsEqual } from '../../../dashboard_api/are_panel_layouts_equal';
-import { useDashboardInternalApi } from '../../../dashboard_api/use_dashboard_internal_api';
-import { DASHBOARD_GRID_HEIGHT, DASHBOARD_MARGIN_SIZE } from '../../../dashboard_constants';
-
-export const DashboardGrid = ({
-  dashboardContainer,
-  viewportWidth,
-}: {
-  dashboardContainer?: HTMLElement;
```

Comment:
Why did you remove `dashboardContainer` props? This value passed to GridItem and used by hover actions to calculate positioning

Reply:
Oops - I think I forgot to add it to `DashboardGridItem` as a prop in `DashboardGrid` and then it was telling me it was unused so I just removed it alongside `viewportWidth` in my cleanup 🤦 Re-added in 822bb967479c79f83950ebae7022304115f4e498 - good catch